src/util/Injected.js

```diff
@@ -263,8 +263,7 @@ exports.LoadUtils = () => {
         const msg = message.serialize();
         
         msg.isStatusV3 = message.isStatusV3;
-        msg.links = (message.getLinks()).map(link => link.href);
-
+        msg.links = (message.getLinks()).map(link => ({ link: link.href, isSus: link.suspiciousCharacters.size ? true : false}));
```

Comment:
Let's use `isSuspicious` instead of `isSus` so it's clearer for the end-user

Reply:
I agree.